project_remaining_time_on_task/models/project.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+from openerp import api, fields, models
+
+
+class Task(models.Model):
+    _inherit = 'project.task'
+
+    remaining_hours_contract = fields.Float(string="Heures restantes sur contrat",
```

Comment:
line too long, try this: 

```python
    remaining_hours_contract = fields.Float(
              string="Heures restantes sur contrat",
              compute='_compute_remaining_hours_contract', 
              digits=(16, 2))
```

Reply:
Plus, correct english would be `remaining_contract_hours` and the `string` field value should be in English as well.

Reply:
ok

Reply:
ok